bundles/core/src/main/java/com/adobe/cq/wcm/core/components/internal/models/v1/NavigationImpl.java

```diff
@@ -85,6 +86,7 @@
     private String navigationRootPage;
     private List<NavigationItem> items;
     private boolean skipNavigationRoot;
```

Comment:
Can we reduce the scope of this variable and only define it locally in `initModel()`?